File: 1672 - Richest Customer Wealth.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize) {
    double max_wealth = 0;
    double customer_wealth;
    for(int i = 0; i < accountsSize; i++){
        customer_wealth = 0;
        for(int j = 0; j < accountsColSize[i]; j++){
            customer_wealth += accounts[i][j];
        }
        if (customer_wealth > max_wealth) max_wealth = customer_wealth;
    }
    return max_wealth;
}

File: 1672 - Richest Customer Wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        number_customers, number_banks = len(accounts), len(accounts[0])
        max_wealth = 0
        for customer_index in range(number_customers):
            customer_wealth = sum(accounts[customer_index])
            max_wealth = max(customer_wealth, max_wealth)
        return max_wealth